website/content/docs/platform/mssql/installation.mdx

```diff
@@ -0,0 +1,190 @@
+
+    path "transit/encrypt/ekm-encryption-key" {
+        capabilities = ["update"]
+    }
+
+    path "transit/decrypt/ekm-encryption-key" {
+        capabilities = ["update"]
+    }
+
+    path "sys/license/status" {
+        capabilities = ["read"]
+    }
+    EOF
+    ```
+
+## Configuring SQL Server
+
+The remaining steps are all run on the database server.
+
+### Install the EKM provider on the server
+
+1. Download and run the latest Vault EKM provider installer from
+  [releases.hashicorp.com](https://releases.hashicorp.com/vault-mssql-ekm-provider/)
+
+1. Enter your Vault server's address when prompted and complete the installer
+
+### Configure the EKM provider using SQL
+
+1. Enable the EKM feature and create a cryptographic provider using the folder
+   you just installed the EKM provider into.
+
+    ```sql
+    -- Enable advanced options
+    USE master;
+    GO
+
+    EXEC sp_configure 'show advanced options', 1;
+    GO
+
+    RECONFIGURE;
+    GO
+
+    -- Enable EKM provider
+    EXEC sp_configure 'EKM provider enabled', 1;
+    GO
+
+    RECONFIGURE;
+    GO
+
+    CREATE CRYPTOGRAPHIC PROVIDER TransitVaultProvider
+    FROM FILE = 'C:\Program Files\HashiCorp\Transit Vault EKM Provider\TransitVaultEKM.dll'
+    GO
+    ```
+
+1. Next, create credentials for an admin to use EKM with your AppRole role and
+    secret ID from above:
+
+    ```sql
+    CREATE CREDENTIAL TransitVaultCredentials
+        WITH IDENTITY = 'c1d62bb3-8238-ccfa-eeb4-7ce01e141d88',
+        SECRET = 'a588486c-aa01-2ead-d0e3-75a1dccaa130'
+    FOR CRYPTOGRAPHIC PROVIDER TransitVaultProvider;
+    GO
+
+    -- Replace <domain>\<login> with the SQL Server administrator's login
+    ALTER LOGIN "<domain>\<login>" ADD CREDENTIAL TransitVaultCredentials;
+    ```
+
+1. You can now create an asymmetric key using the transit key set up earlier:
+
+    ```sql
+    CREATE ASYMMETRIC KEY TransitVaultAsymmetric
+    FROM PROVIDER TransitVaultProvider
+    WITH
+    CREATION_DISPOSITION = OPEN_EXISTING,
+    PROVIDER_KEY_NAME = 'ekm-encryption-key';
+    ```
+
+1. Create another login from the new asymmetric key:
+
+    ```sql
+    CREATE CREDENTIAL TransitVaultTDECredentials
+        WITH IDENTITY = 'c1d62bb3-8238-ccfa-eeb4-7ce01e141d88',
+        SECRET = 'a588486c-aa01-2ead-d0e3-75a1dccaa130'
+    FOR CRYPTOGRAPHIC PROVIDER TransitVaultProvider;
+    GO
+
+    CREATE LOGIN TransitVaultTDELogin
+    FROM ASYMMETRIC KEY TransitVaultAsymmetric;
+    GO
+
+    ALTER LOGIN TransitVaultTDELogin
+    ADD CREDENTIAL TransitVaultTDECredentials;
+    GO
+    ```
+
+1. Finally, you can enable TDE and protect the database encryption key with
+   the asymmetric key managed by Vault's Transit secret engine:
+
+    ```sql
+    CREATE DATABASE TestTDE
+    GO
+
+    USE TestTDE;
+    GO
+
+    CREATE DATABASE ENCRYPTION KEY
+    WITH ALGORITHM = AES_256
+    ENCRYPTION BY SERVER ASYMMETRIC KEY TransitVaultAsymmetric;
+    GO
+
+    ALTER DATABASE TestTDE
+    SET ENCRYPTION ON ;
```

Comment:
Is the space before the semicolon intentional?

Reply:
Nope! Good spot :)